io/EptReader.cpp

```diff
@@ -653,6 +656,36 @@ uint64_t EptReader::readBinary(PointView& dst, const Key& key,
     return startId;
 }
 
+uint64_t EptReader::readZstandard(PointView& dst, const Key& key,
+        const uint64_t nodeId) const
+{
+    auto compressed(m_ep->getBinary("ept-data/" + key.toString() + ".zst"));
+    std::vector<char> uncompressed;
+    pdal::ZstdDecompressor dec([&uncompressed](char* pos, std::size_t size)
+    {
+        uncompressed.insert(uncompressed.end(), pos, pos + size);
+    });
+
+    dec.decompress(compressed.data(), compressed.size());
+
+    auto tab = std::move(uncompressed);
+    ShallowPointTable table(*m_remoteLayout,tab.data(), tab.size());
+    PointRef pr(table);
+
+    std::lock_guard<std::mutex> lock(m_mutex);
+
+    const uint64_t startId(dst.size());
```

Comment:
I guess we should update the whole file to use PointId instead of uint64_t. I'll do that once the zstandard reader has got the correct behaviour. I'll also wait for the otehr PR on the same file to be merged. #2562